modules/seqexec/server/src/main/scala/seqexec/server/altair/AltairControllerEpics.scala

```diff
@@ -73,14 +73,14 @@ object AltairControllerEpics extends AltairController[IO] {
   implicit val fieldLensEq: Eq[FieldLens] = Eq.by(_.ordinal)
 
   private def pauseNgsOrLgsMode(starPos: (Length, Length), fieldLens: FieldLens, currCfg: EpicsAltairConfig)(
-    reasons: Set[Gaos.PauseCondition])
+    reasons: Set[PauseCondition])
   : Option[(EpicsAltairConfig, IO[Unit])] = {
-    val offsets = reasons.collectFirst { case OffsetMove(p, n) => (p, n) }
+    val offsets = reasons.collectFirst { case PauseCondition.OffsetMove(p, n) => (p, n) }
```

Comment:
PauseConditionn cases were moved to the companion object.

Reply:
"That won't happen" :)

Reply:
Now seriously, it won't happen because there is one offset per step. I see your point, though I am not sure how to tighten the data type.

Reply:
I'm not sure of the context, but could it be that you need an `Option[OffsetMove]` and a `Set[OtherPauseConditions]`.  By `OtherPauseConditions` I mean something that includes all the `PauseCondition`s that you currently have but not `OffsetMove`.

Reply:
I could create a specialized `PauseConditions` to use instead of `Set[PauseConditions]`. Something like `case class PauseConditions(offsetO: Option[OffsetMove], other: Set[OtherPauseCondition])`, and with some convenience functions to put and retrieve values.

Reply:
Yes, I think that makes sense with an appropriate name for `OtherPauseCondition` presumably.